03-task-manager/starter/app.js

```diff
@@ -1,19 +1,29 @@
-require('dotenv').config();
 const express = require('express');
 const app = express();
 const tasks = require('./routes/tasks');
 const connectDB = require('./db/connect');
+require('dotenv').config();
```

Comment:
Nitpick:
I'd recommend moving this `require('dotenv').config();` statement to the top of the file. If you were to have a file that invokes a function which uses an ENV, then on import/require I believe the ENV will be undefined.

Reply:
Yes, you are right, that was on top on previous week, but because I 've seen on the video, it has been moved in the middle, I did the same. Thank you for pointing it out.